ntbs-service/DataAccess/NotificationRepository.cs

```diff
@@ -307,8 +307,13 @@ public async Task<IEnumerable<NotificationBannerModel>> GetNotificationBannerMod
                         Sex = n.PatientDetails.SexLabel,
                         NotificationStatus = n.NotificationStatus,
                         DrugResistance = n.DrugResistanceProfile.DrugResistanceProfileString,
-                        TreatmentEvents = n.TreatmentEvents
+                        TreatmentEvents = n.TreatmentEvents,
+                        PreviousTbServiceCodes = n.PreviousTbServices.Select(service => service.TbServiceCode),
+                        PreviousPhecCodes = n.PreviousTbServices.Select(service => service.PhecCode),
+                        LinkedNotificationPhecCodes = n.Group.Notifications.Select(no => no.HospitalDetails.TBService.PHECCode),
+                        LinkedNotificationTbServiceCodes = n.Group.Notifications.Select(no => no.HospitalDetails.TBServiceCode)
                     })
+                .AsSingleQuery()
```

Comment:
To avoid this error: 
![image](https://user-images.githubusercontent.com/78739678/130035476-d0d97dd7-9535-4aed-b92b-050ac04add43.png)

Reply:
Interesting! Do you know which collection cannot be split?

I definitely think it would be worth adding a comment here to explain why you've put this line in.

Reply:
Both collections` n.PreviousTbServices` and `n.Group.Notifications` will throw this error. Will write a comment